examples/ili9341/pyportal_boing/main.go

```diff
@@ -58,6 +58,7 @@ func main() {
 	backlight.High()
 
 	display.SetRotation(ili9341.Rotation270)
+	time.Sleep(50 * time.Millisecond)
```

Comment:
Any reasons for the addition?

Reply:
Unfortunately I don't have any documentation to justify this, only a use case:

Before the boing demo enters its main loop (drawing the ball frames), it first draws the entire 1-bit background across the screen. 

With my Feather, the top ~0.5cm of this background (including the top of the grid) was not being drawn, and would remain either white or have a scrambled gray coloring -- it seemed like the first SPI transactions were starting too quickly and the screen wasn't able to draw the data in time. The rest of the graphics continued normally without any problem. 

After adding this short delay, I never saw the problem again. 

I'll try to recreate and take a screenshot.